### ISSUE TITLE
Layout Change in Signup Screen

### ISSUE THREAD

**Issue by akkywadhwa** [openMF/mobile-wallet#571]
**Summary:**
1. Resizing the `TextInputEditText` 
2. Changing the color of back button  
3. Resizing the control being used for selecting country

**Steps to reproduce:**
Open the app and click "Create One" textview located at bottom.

**Expected behavior:**
![edited](https://user-images.githubusercontent.com/666752/69886334-73fed200-1307-11ea-82e9-fd9feb8ffa40.PNG)
Please ignore the background as this is an edited screenshot of app to represent the idea.

**Observed behavior:**
![original](https://user-images.githubusercontent.com/666752/69886351-8da01980-1307-11ea-9775-018a71047fb6.PNG)

**Device and Android version:**
All devices

**Screenshots:**
Attached above.

**Comment by jsahil730** [comment]
The green tick doesn't look good, how or where do you suppose you'll keep the 'grey arrow' button to submit mobile number? @akkywadhwa

**Comment by akkywadhwa** [comment]
I think I missed to place the button correctly in the above screenshot. Here is the revised screenshot. 
![edited](https://user-images.githubusercontent.com/666752/70601388-ed57c800-1c17-11ea-9d0b-b09f51a679a4.PNG)

**Comment by jsahil730** [comment]
@luckyman20 Please review this. I recommend also removing the boundary around the 'Enter OTP' edit text to make it look better.

**Comment by luckyman20** [comment]
@jsahil730 the boundary around is part of MaterialEditText, what we can do is we can bring it a little down or shift it to another fragment or we have re-design this screen (recommended). 

If anyone of you is interested in doing so then please go ahead.

**Comment by jsahil730** [comment]
@luckyman20 I'd like it to be re-designed, but how do you think it should look? Anything that should be added or removed?

**Comment by akkywadhwa** [comment]
@luckyman20 I was already doing this. Will make a PR for this soon.

**Comment by luckyman20** [comment]
@jsahil730 I don't we need any additions or deletions but we can re-arrange the views and make it look proper.

@akkywadhwa I would recommend to first post the mockup here before implementing it.

**Comment by akkywadhwa** [comment]
> 
> 
> @jsahil730 I don't we need any additions or deletions but we can re-arrange the views and make it look proper.
> 
> @akkywadhwa I would recommend to first post the mockup here before implementing it.



> 
> 
> I think I missed to place the button correctly in the above screenshot. Here is the revised screenshot.
> ![edited](https://user-images.githubusercontent.com/666752/70601468-15dfc200-1c18-11ea-9603-fa2d93895b05.PNG)
> Also, on entering the number the button might blink to make user aware about the button to be pressed. This will certainly enhance the user experience and make the application self explanatory.

@luckyman20 this is the UI mockup I have proposed.

**Comment by luckyman20** [comment]
@akkywadhwa This doesn't looks very intuitive if you see through the eyes of user.

Since GCI season is going on, @rachittt @laxyapahuja @OussEmaDevCode do anyone of you want to help?

**Comment by jsahil730** [comment]
@luckyman20 the button blinking idea is very good tbh

**Comment by rachittt** [comment]
@luckyman20 yup, can i work on it ?

**Comment by luckyman20** [comment]
@rachittt Yes, just send mockup as of now.

**Comment by rachittt** [comment]
@luckyman20 okay

**Comment by harshshinde07** [comment]
@luckyman20 Here's the UI mockup I'm proposing.

<img src="https://user-images.githubusercontent.com/25999879/70623146-7b957380-1c43-11ea-998e-90f4f192a24e.png" width=350 />

The back button on actionbar and next icon on fab should be in white, I missed it.

Also the dropdown doesn't have any border, it's just that the mockup was setting the same border to dropdown icon, thus making it invisible.

**Comment by rachittt** [comment]
Below are the designs that i would like to propose:-

Entering Mobile Number-
![Android Mobile – 9](https://user-images.githubusercontent.com/44428198/70739502-00b08380-1d3d-11ea-8b3d-ad381779a846.png)

Entering OTP-
![Android Mobile – 11](https://user-images.githubusercontent.com/44428198/70739509-0312dd80-1d3d-11ea-86b3-b8baaedbdceb.png)

creating different fragments looks like a good option to me, what do you all think ?

**Comment by luckyman20** [comment]
@rachittt This looks good to me.

@akkywadhwa Since you opened the issue, do you want to work on this?

**Comment by akkywadhwa** [comment]
> 
> 
> @rachittt This looks good to me.
> 
> @akkywadhwa Since you opened the issue, do you want to work on this?

Yes, @luckyman20 I would be happy to work on this.

**Comment by akkywadhwa** [comment]
> 
> 
> Below are the designs that i would like to propose:-
> 
> Entering Mobile Number-
> ![Android Mobile – 9](https://user-images.githubusercontent.com/44428198/70739502-00b08380-1d3d-11ea-8b3d-ad381779a846.png)
> 
> Entering OTP-
> ![Android Mobile – 11](https://user-images.githubusercontent.com/44428198/70739509-0312dd80-1d3d-11ea-86b3-b8baaedbdceb.png)
> 
> creating different fragments looks like a good option to me, what do you all think ? @jsahil730 @luckyman20 @harshshinde07 @akkywadhwa

I hope we have finalized to stick with this UI. Right?

**Comment by luckyman20** [comment]
@akkywadhwa Yes, we are sticking to this UI only. We will send 6-digit OTP but we don't API for that as of now so let it be.

**Comment by ReCodee** [comment]
@devansh-299 can I work on this issue?

**Comment by therajanmaurya** [comment]
Fixed in compose implementation
